frontend/src/component/common/SidePanelList/SidePanelList.tsx

```diff
@@ -43,12 +44,14 @@ export const StyledSidePanelListColumn = styled('div', {
     shouldForwardProp: (prop) => prop !== 'maxWidth' && prop !== 'align',
 })<{ maxWidth?: number; align?: ColumnAlignment }>(
     ({ theme, maxWidth, align = 'start' }) => ({
+        display: 'flex',
         flex: 1,
         padding: theme.spacing(2),
         fontSize: theme.fontSizes.smallBody,
         justifyContent: align,
         ...(maxWidth && { maxWidth }),
         textAlign: align,
+        alignItems: 'center',
```

Comment:
Will this and the display change above affect other UI components? I'm just worried this might cause unnoticed changes a few days prior a release

Reply:
`SidePanelList` is a new component introduced in https://github.com/Unleash/unleash/pull/6317 so I wouldn't worry about it.

The only other place where we're using it, Incoming webhook events, also looks correct with this change.